src/libraries/System.Private.Xml/src/System/Xml/XmlQualifiedName.cs

```diff
@@ -36,83 +34,69 @@ public class XmlQualifiedName
         /// </devdoc>
         public XmlQualifiedName(string? name, string? ns)
         {
-            _ns = ns ?? string.Empty;
-            _name = name ?? string.Empty;
+            Namespace = ns ?? string.Empty;
+            Name = name ?? string.Empty;
         }
 
         /// <devdoc>
         ///    <para>[To be supplied.]</para>
         /// </devdoc>
-        public string Namespace
-        {
-            get { return _ns; }
-        }
-
+        public string Namespace { get; private set; }
         /// <devdoc>
         ///    <para>[To be supplied.]</para>
         /// </devdoc>
-        public string Name
-        {
-            get { return _name; }
-        }
-
+        public string Name { get; private set; }
```

Comment:
```suggestion
        public string Name { get; }
```

Reply:
Ah, nevermind, I see we set it later outside of the constructor.

Reply:
see internal void Init(string? name, string? ns)